src/app/(providers)/(root)/detail/[id]/page.tsx

```diff
@@ -1,5 +1,49 @@
-const DetailPage = () => {
-  return <div>DetailPage</div>;
+'use client';
+import { useParams } from 'next/navigation';
+import { getRecipeById } from '@/lib/fetchData';
+import { IconBackButton } from '@/icons/IconBackButton';
+import { IconSearch } from '@/icons/IconSearch';
+import { Recipe } from '@/types/recipe';
+import { useQuery, UseQueryResult } from '@tanstack/react-query';
+import RecipeSection from '@/app/(providers)/(root)/detail/_components/RecipeSection';
```

Comment:
import 경로에 오류가 있는 것 같습니다.
```suggestion
import RecipeSection from '@/app/detail/_components/RecipeSection';
```
혹시 이 경로로 안되실까요?

Reply:
음 네 저는 안되는데 ㅠㅠ 설정 문제일까요??